bats/core/api/public-ln-receive.bats

```diff
@@ -110,6 +110,56 @@ usd_amount=50
   [[ "$error_msg" == "Account does not exist for username idontexist" ]] || exit 1
 }
 
+@test "public-ln-receive: account details - can fetch default wallet with phone number" {
```

Comment:
you need to test default, btc and usd... also this name is a bit confusing because it says default but you are testing BTC wallet